doc/developers/bug_triaging.rst

```diff
@@ -0,0 +1,142 @@
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_
+is important to the communication in the project: it helps
+developers identify major projects to work on, as well as to discuss
+priorities. For this reason, it is important to curate it, adding labels
+to issues and closing issues that are not necessary.
+
+Working on issues to improve them
+--------------------------------------
+
+Improving issues increases their chances of being successfully resolved.
+Guidelines on submitting good issues can be found :ref:`here
+<filing_bugs>`. A third party can give useful feedback or even add
+comments on the issue, while core-developpers or members of the triage
+team can edit the issue description and title.
+
+The following actions are typically useful:
+
+  - documenting issues that are missing elements to reproduce the problem
+    such as code samples
+
+  - correcting incorrect use of code formatting
+
+  - making sure that the title and description are explicit about the
+    problem to be solved
+
+  - linking to related issues or discussions
```

Comment:
Often we just write "See also #xyz" and it's then up to everyone else to figure out how the issues are related. "See also #xyz for a similar attempt at this" or "See also #xyz where the same thing happened in SomeEstimator" provide context and are often more useful.

```suggestion
  - linking to related issues or discussions while briefly describing how they are related 
```